index.js

```diff
@@ -59,10 +59,7 @@ function getExtractedSVG(svgStr, query) {
 function SVGInlineLoader(content) {
     this.cacheable && this.cacheable();
     this.value = content;
-    // Configuration
-    var query = loaderUtils.parseQuery(this.query);
-
```

Comment:
This should still be required. `var query = loaderUtils.getOptions(this) || {};`